src/test/java/org/openstreetmap/atlas/utilities/command/subcommands/AtlasSearchCommandTest.java

```diff
@@ -283,6 +282,51 @@ public void testInOutEdgeSearch()
         }
     }
 
+    @Test
+    public void testJsonOption()
+    {
+        try (FileSystem filesystem = Jimfs.newFileSystem(Configuration.osX()))
+        {
+            setupFilesystem1(filesystem);
+            final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+            final ByteArrayOutputStream errContent = new ByteArrayOutputStream();
+            final AtlasSearchCommand command = new AtlasSearchCommand();
+            command.setNewFileSystem(filesystem);
+            command.setNewOutStream(new PrintStream(outContent));
+            command.setNewErrStream(new PrintStream(errContent));
+
+            command.runSubcommand("/Users/foo/test.atlas", "--verbose", "--all", "--json");
+
+            Assert.assertEquals(
+                    "{\"shard\":\"unknown_unknown\",\"path\":\"/Users/foo/test.atlas\",\"entity\":{\"identifier\":8000000,\"type\":\"NODE\",\"geometry\":\"POINT (30 30)\",\"tags\":{\"foo\":\"bar\"},\"inEdges\":[],\"outEdges\":[11000000],\"parentRelations\":[],\"bounds\":\"POLYGON ((30 30, 30 30, 30 30, 30 30, 30 30))\"}}\n"
```

Comment:
It would be nice to have that in a resource file, and then load it for the test!